python-main.js

```diff
@@ -1356,11 +1358,11 @@ function web_editor(config) {
             // Stop closure of the menu in other local event handlers
             e.originalEvent.keepMenuOpen = true;
         });
-
         $(".lang-choice").on("click", function() {
             $("#language_container").addClass('hidden');
             TRANSLATIONS.updateLang($(this).attr('id'), function(translations) {
                 config.translate = translations;
+            document.getElementsByTagName("HTML")[0].setAttribute("lang", lang);
```

Comment:
This line should be indented as it goes inside the anonymous function.

```suggestion
                document.getElementsByTagName("HTML")[0].setAttribute("lang", lang);
```